src/ctl/plugins/version_base.py

```diff
@@ -117,6 +123,11 @@ def add_arguments(cls, parser, plugin_config, confu_cli_args):
             choices=["major", "minor", "patch", "prerelease"],
             help="bumps the specified version segment by 1",
         )
+        op_bump_parser.add_argument(
+            "--do_not_tag",
```

Comment:
@solababs nitpick - should be `--do-not-tag` or even better `--no-tag`

Also needs to be updated in docs: https://github.com/20c/ctl/blob/main/docs/plugins/version.md